setup.cfg

```diff
@@ -8,9 +8,9 @@ long_description = file: README.md
 long_description_content_type = text/markdown
 classifiers =
     Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
     Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
```

Comment:
drop Python 3.9 sport in favor of 3.12 is not part of this PR description.
I'm not opposed to the change, but it should be mentioned on the PR description.

Reply:
done